tests/docker-build.ts

```diff
@@ -184,12 +182,13 @@ export async function testDockerBuildStream(o: {
 
 		o.dockerMock.expectPostBuild({
 			...o,
-			checkURI: async (uri: string) => {
+			checkURI: (uri: string) => {
 				const url = new URL(uri, 'http://test.net/');
 				const queryParams = Array.from(url.searchParams.entries());
 				expect(deepJsonParse(queryParams)).to.have.deep.members(
 					deepJsonParse(expectedQueryParams),
 				);
+				return Promise.resolve();
```

Comment:
Why `Promise.resolve`?

Reply:
The expected interface (which I would rather not change on a linter bump) expects a promise. Beforehand we were using the syntax of a `async` function without doing any async work inside of it. The linter no longer likes this syntax and prefer the more explicit `Promise.resolve()` for a `Promise<void>` type